docs/Software Architecture Document/Components View/index.md

```diff
@@ -13,135 +13,150 @@ The view is represented by module and subsystem diagrams that show the system's
+  - *Responsibility*
+    -    Storing information about application configuration.
   - *Relations*
-    - Used by ***Life Cycle*** for filling other components ***Settings *** 
-  - *Interfaces* 
-    - Provides ***Profile*** interface 
-  - *Behavior* 
-    - ***Config Profile*** parses configurable data storage and provides primitive types by section and name of configurable value. 
-  - *Constraints* 
-    - Configuration format - INI file. 
+    - Used by ***Life Cycle*** for filling other components ***Settings ***
+  - *Interfaces*
+    - Provides ***Profile*** interface
+  - *Behavior*
+    - ***Config Profile*** parses configurable data storage and provides primitive types by section and name of configurable value.
+  - *Constraints*
+    - Configuration format - INI file.
 
 #### Utils
-  - *Responsibility* 
-    - Encapsulation system low-level functionality. 
-  - *Relations* 
-    -   Used by all components. 
-  - *Interfaces* 
-    - ***Logger*** macros-es and functions 
+  - *Responsibility*
+    - Encapsulation system low-level functionality.
+  - *Relations*
+    -   Used by all components.
+  - *Interfaces*
+    - ***Logger*** macros-es and functions
     - Data and Time
     - Files
     - ***Thread*** and ***Timer***
     - ***Locks*** and ***ConditionalVariable*** classes
-    - ***CustomString*** class for UTF8 string handling 
+    - ***CustomString*** class for UTF8 string handling
   - *Behavior*
-    - ***Utils*** behavior relates to system-specific API. 
+    - ***Utils*** behavior relates to system-specific API.
   - *Constraints*
-    - *N/A* 
+    - *N/A*
 
 ### HMI layer components:
 
 #### HMI Message Handler
   - *Responsibility*
     - Formatting message to and from unified protocol-API-independent format used by higher-level component.
-    - Providing adapters for different transport types between SDL and HMI. 
+    - Providing adapters for different transport types between SDL and HMI.
   - *Relations*
     - ***Application Manager***
-    - ***Utils*** 
-  - *Interfaces* 
+    - ***Utils***
+  - *Interfaces*
     - ***HMIMessageObserver*** interface for listening HMI messages notification
     - ***HMIMessageSender*** interface for sending Messages
     - ***HMIMessageAdapter*** interface for abstracting to-HMI transport
-    - ***HMIMessageHandler*** interface for accumulating ***HMIMessageObserver, HMIMessageSender*** and ***HMIMessageAdapter*** 
+    - ***HMIMessageHandler*** interface for accumulating ***HMIMessageObserver, HMIMessageSender*** and ***HMIMessageAdapter***
   - *Behavior*
-    - Transferring RPC Messages between business-layer and configured transport. 
-  - *Constraints* 
+    - Transferring RPC Messages between business-layer and configured transport.
+  - *Constraints*
     - Processes messages from a single instance of HMI only.
     - HMI-transport need to be statically configurable with build flags.
 
 ### Business layer components:
```

Comment:
in the diagram, it is called `application` layer, not `business` layer, and there are 10 components in the diagram, but 12 components in this list. Something mismatch?

Reply:
@yang1070 fixed 3f6450a